packages/server/src/api/routes/utils/validators.ts

```diff
@@ -88,7 +91,7 @@ export function datasourceValidator() {
   )
 }
 
-function filterObject() {
+function filterObject(unknown = true) {
```

Comment:
```suggestion
function filterObject(opts?: { unknown: boolean }) {
  const { unknown = true } = opts || {}
```

Reply:
This just makes it a little bit easier to understand whehn this function is called, what this boolean is actually doing or referring to.